website/content/en/docs/Community/Governance/_index.md

```diff
@@ -0,0 +1,90 @@
+weight: 10
+---
+
+<!-- The governance model is similar to https://github.com/containerd/project/blob/main/GOVERNANCE.md but simplified -->
+
+## Code of Conduct
+Lima follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Maintainership
+Lima is governed by Maintainers who are elected from active contributors.
+
+As a [Cloud Native Computing Foundation](https://cncf.io/) project, Lima will keep its [vendor-neutrality](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).
+
+### Roles
+Maintainers consist of two roles:
+
+- **Committer** (Full maintainership): Committers have full write accesses to repos under <https://github.com/lima-vm>.
+  Committers' commits should still be made via GitHub pull requests (except for urgent secuirity fixes), and should not be pushed directly.
+  Committers are also recognized as Maintainers in <https://github.com/cncf/foundation/blob/main/project-maintainers.csv>.
+
+- **Reviewer** (Limited maintainership): Reviewers may moderate GitHub issues and pull requests (such as adding labels and cleaning up spams),
+  but they do not have any access to merge pull requests nor push commits.
+  A Reviewer is considered as a candidate of Committers.
+  Reviewers are not recognized as Maintainers in <https://github.com/cncf/foundation/blob/main/project-maintainers.csv>.
+
+See also the [Contributing](../contributing) page.
+
+### Current maintainers
+
+| Name               | Role      | GitHub ID (not X ID)                           | GPG fingerprint                                                                          |
+|--------------------|-----------|------------------------------------------------|------------------------------------------------------------------------------------------|
+| Akihiro Suda       | Committer | [@AkihiroSuda](https://github.com/AkihiroSuda) | [C020 EA87 6CE4 E06C 7AB9  5AEF 4952 4C6F 9F63 8F1A](https://github.com/AkihiroSuda.gpg) |
+| Jan Dubois         | Committer | [@jandubois](https://github.com/jandubois)     |                                                                                          |
+| Anders F Björklund | Committer | [@afbjorklund](https://github.com/afbjorklund) |                                                                                          |
+| Balaji Vijayakumar | Committer | [@balajiv113](https://github.com/balajiv113)   |                                                                                          |
+
+<!-- TODO: invite non-committer reviewers -->
+
+### Addition and promotion of Maintainers
+An active contributor to the project can be invited as a Reviewer,
+and can be eventually promoted to a Committer after 2 months at least.
+
+A contributor who have made significant contributions in quality and in quantity
+can be also directly invited as a Committer.
+
+A proposal to add or promote a Maintainer must be approved by 2/3 of the Committers who vote within 7 days.
```

Comment:
Should there be a quorum for any of the voting, or is a single vote enough if nobody else votes in the time period?

Reply:
Maybe it should need 2 approvals (including the proposer) at least?

Reply:
Added "Voting needs 2 approvals at least. The proposer can vote too."